website/docs/dev_colorspace.md

```diff
@@ -0,0 +1,113 @@
+---
+id: dev_colorspace
+title: Colorspace Management and Distribution
+sidebar_label: Colorspace
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Introduction
+Defines the distribution of colors and OCIO config during publishing. Once colorspace data are captured and integrated into representation loaders could use them for loading image and video data with correct colorspace.
+
+:::warning Color Management (ImageIO)
+Settings schema is requred for any host or module which is processing pixel data.
+:::
+
+## Data model
+The *colorspaceData* are stored at root of representation dictionary during publishing. Once they are integrated into representation db document they are stored as *representation_doc.data["colorspaceData"]*
+
+### Keys
+- **colorspace** - string value used in other publish plugins and loaders
+- **configData** - storing two versions of path.
```

Comment:
Yeah, I agree. Not sure `Data` is needed, but also why are we storing two paths?
`configData` could just be a string/path which always needs formatting to ensure its absolute.

Reply:
No issue with renaming `configData` to `config` key. But my recommendation is to keep absolute path in the config for future reference in case something goes wrong. The same way as we store path in representations db document.

Reply:
Sounds good. Anything wrong with naming the entries `absolute` and `relative`?

Reply:
One technically isn't relative but a template. And the absolute path might not be a valid absolute on your current machine if you're in a remote location or another OS. 

The `template` I think is nice. The `path` or `absolute` entries I'm not sure about. I would avoid giving to explicit of a meaning to it.